brownie/network/contract.py

```diff
@@ -451,7 +451,7 @@ def publish_source(self, contract: "Contract", silent: bool = False) -> bool:
             "compilerversion": f"v{contract_info['compiler_version']}",
             "optimizationUsed": 1 if contract_info["optimizer_enabled"] else 0,
             "runs": contract_info["optimizer_runs"],
-            "constructorArguements": constructor_arguments,
+            "constructorArguments": constructor_arguments,
```

Comment:
How has this one never caused issues for anybody before?

Reply:
apparently this typo is supposed to be there... 
https://docs.etherscan.io/etherscan-v2/common-verification-errors